ajax/libs/jquery.scrollfire/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.scrollfire",
+  "filename": "jquery.scrollfire.min.js",
+  "version": "1.4.0",
+  "description": "Allows useful callbacks to be fired upon elements scrolling into and out of view from both the top and bottom of the viewport/window.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Xaxis/jquery.scrollfire.git"
+  },
+  "keywords": [
```

Comment:
You can add keywords from upstream. Take a look at https://github.com/Xaxis/jquery.scrollfire/blob/master/scrollfire.jquery.json#L6-L8